src/test/scala/com/supersonic/datadog/JSONTestUtil.scala

```
@@ -5,10 +5,9 @@ import io.circe.parser.parse
 
 object JSONTestUtil {
 
-  def unsafeParse(json: String): Json =
-    parse(json).right.get // because it's a test and guaranteed to succeed
+  def unsafeParse(json: String): Json = parse(json).getOrElse(Json.Null) // because it's a test and guaranteed to succeed
```

I'd actually prefer to throw here rather than succeed with `null`